lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -589,9 +672,11 @@ void ProxyRenderDelegate::_Execute(const MHWRender::MFrameContext& frameContext)
     if (inSelectionPass && !inPointSnapping) {
         _globalListAdjustment = GetListAdjustment();
         _selectionKind = GetSelectionKind();
+        _pointInstancesPickMode = GetPointInstancesPickMode();
     } else {
         _globalListAdjustment = MGlobal::kReplaceList;
         _selectionKind = TfToken();
+        _pointInstancesPickMode = UsdPointInstancesPickMode::PointInstancer;
```

Comment:
Did you test the behavior of point snapping on point instances? What is the expected behavior?

Reply:
I check the getInstancedSelectionPath() implementation. This value shouldn't matter for point snapping. What matters for point snapping is the hit points which will be calculated by VP2.

Reply:
Admittedly, no, I did not test point snapping against a point instancer. I'm not sure whether I'm interpreting your suggestion correctly though. Are you saying we don't need to set `_pointInstancesPickMode` in the `else` case here and this should be removed?